src/ess/imaging/normalize.py

```diff
@@ -190,21 +190,18 @@ def apply_threshold_to_sample_images(
 
     sample_threshold:
         Threshold for the sample pixel values.
-        Any pixel values less than ``sample_threshold``
-        are replaced with ``sample_threshold``.
+        Any pixel values less than ``sample_threshold`` will be masked.
 
     """
     samples = CleansedSampleImages(samples.copy(deep=False))
-    samples.data = sc.where(
-        samples.data < sample_threshold, sample_threshold, samples.data
-    )
+    samples.masks['counts'] = samples.data < sample_threshold
     return SampleImageStacks(samples)
```

Comment:
How about just using something like
```Py
return SampleImageStacks(samples.assign_masks(counts=samples.data < sample_threshold))
```
?
This would remove the need for the manual copy above.